test/unit/internal/test_tuf.py

```diff
@@ -120,6 +124,44 @@ def test_updater_staging_get(mock_staging_tuf, tuf_asset):
         assert updater.get_rekor_keys() == [f.read()]
 
 
+def test_bundled_get(monkeypatch, mock_prod_tuf, asset):
+    def _contents(name: str):
+        path = Path("prod-tuf") / name
+        return asset(str(path)).read_bytes()
+
+    def _der_keys(keys):
+        return [
+            load_der_public_key(k).public_bytes(
+                Encoding.DER, PublicFormat.SubjectPublicKeyInfo
+            )
+            for k in keys
+        ]
+
+    def _pem_keys(keys):
+        return [
+            load_pem_public_key(k).public_bytes(
+                Encoding.DER, PublicFormat.SubjectPublicKeyInfo
+            )
+            for k in keys
+        ]
+
+    updater = TrustUpdater.production()
+
+    # The test should use the TUF path, so we stub out the legacy getter here.
+    monkeypatch.setattr(updater, "_get", lambda usage, statuses: [])
+
+    assert _der_keys(updater.get_ctfe_keys()) == _pem_keys([_contents("ctfe_2022.pub")])
```

Comment:
Just so I know what to expect. When there's a key rotation, we should expect these to begin failing right? And in that case, we should refresh the keys and certs under `assets/prod-tuf`.

Reply:
Hmm, I'm not too bothered by it. If it becomes annoying then we can look into monkey patching the time, but for now I'd say we can switch to staging assets when it's ready and pull the trigger on this.